src/Toast/Toast.stories.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import { Story, Meta } from '@storybook/react'
+
+import Toast, { ToastProps } from '.'
+import Alert from '../Alert'
+import Button from '../Button'
+
+export default {
+  title: 'Layout/Toast',
+  component: Toast,
+} as Meta
+
+export const Default: Story<ToastProps> = (args) => {
+  return (
+    <div className="w-full h-full">
+      <Toast {...args}>Default toast</Toast>
```

Comment:
For these stories without any content being displayed inside the `preview`, can we add some text like "Look to the bottom right of the screen" or something along those lines to prompt the user (and make it seem less empty)?

Reply:
@benjitrosch added some dynamic text for these